src/main/java/svc/data/citations/filters/CitationFilter.java

```diff
@@ -11,25 +12,45 @@
 import svc.managers.CourtManager;
 import svc.models.Citation;
 import svc.models.Court;
+import svc.models.Violation;
 
 @Component
 public class CitationFilter {
 	@Inject
 	CourtManager courtManager;
 
 	public List<Citation> RemoveCitationsWithExpiredDates(List<Citation> citations){
```

Comment:
Now that this method doesn't just remove citations with expired dates, maybe this method needs to be renamed to something more generic. Maybe just "Filter"? CitationFilter.filter() - which fits more of an interface style because then the CitationFilter decides what needs to be done to filter and the outside world doesn't have to care.